src/app/modules/workspace/__tests__/workspace.service.spec.ts

```diff
@@ -275,4 +278,181 @@ describe('workspace.service', () => {
       expect(actual._unsafeUnwrapErr()).toBeInstanceOf(WorkspaceNotFoundError)
     })
   })
+
+  describe('deleteWorkspace', () => {
+    const mockFormId = new ObjectId()
+    const mockWorkspace = {
+      _id: new ObjectId(),
+      admin: new ObjectId(),
+      title: 'workspace1',
+      formIds: [mockFormId],
+      count: 0,
+    }
+
+    it('should return undefined when successfully deleted workspace', async () => {
+      jest.spyOn(WorkspaceModel, 'deleteWorkspace').mockResolvedValueOnce(true)
+
+      const actual = await WorkspaceService.deleteWorkspace({
+        workspaceId: mockWorkspace._id.toHexString(),
+        shouldDeleteForms: false,
+      })
+
+      expect(actual.isOk()).toEqual(true)
+      expect(actual._unsafeUnwrap()).toEqual(undefined)
+    })
+
+    it('should return undefined when failed to delete workspace', async () => {
```

Comment:
this illustrates the previous comment - we return the same thing regardless of the outcome of the deletion